src/emulator/host/include/host/get_sfo.h

```diff
@@ -22,9 +22,9 @@
 
 struct SfoFile;
 
-namespace app {
+namespace sfo {
 
-bool load_sfo(SfoFile &file, const std::vector<uint8_t> &data);
+bool load(SfoFile &sfile, const std::vector<uint8_t> &data);
```

Comment:
Recommend a better name for this file

Reply:
Ended up merging the sfo files into one as well.